test/extension.test.js

```diff
@@ -61,7 +61,7 @@ describe.skip('Test sidekick extension', () => {
             prod: {
               host: 'www.example.com',
             },
-            contentSourceUrl: 'https://example.sharepoint.com/documents/website',
+            mountpoints: 'https://example.sharepoint.com/documents/website',
```

Comment:
This should be an array

Reply:
fixed in [e5b214b](https://github.com/adobe/helix-sidekick-extension/pull/727/commits/e5b214b202d1e462982b19e00ef5de3a2696d022)